test/validation-controller-factory.ts

```diff
@@ -13,4 +14,16 @@ describe('ValidationControllerFactory', () => {
     expect(container.get(Optional.of(ValidationController))).toBe(null);
     expect(childContainer.get(Optional.of(ValidationController))).toBe(controller);
   });
+
+  it('createForCurrentScopeWithValidator', () => {
+    const container = new Container();
+    const childContainer = container.createChild();
+
+    const factory = childContainer.get(ValidationControllerFactory);
+    const validator = childContainer.get(Validator);
```

Comment:
I think the test would be more clear if this line was changed to something like this:
```js
const validator = {};
```

Otherwise, `const controller = factory.createForCurrentScope();` would also pass the `expect(controller.validator).toBe(validator)` test.